provider/go.mod

```diff
@@ -6,60 +6,73 @@ replace (
 	github.com/hashicorp/go-getter v1.5.0 => github.com/hashicorp/go-getter v1.4.0
 	github.com/hashicorp/terraform-plugin-sdk/v2 => github.com/pulumi/terraform-plugin-sdk/v2 v2.0.0-20220725190814-23001ad6ec03
 	github.com/minio/minio => github.com/minio/minio v0.0.0-20210409155609-0ddc4f00756b
-	github.com/minio/minio-go/v7 => github.com/minio/minio-go/v7 v7.0.11-0.20210302210017-6ae69c73ce78
+	github.com/minio/minio-go/v7 => github.com/minio/minio-go/v7 v7.0.47
```

Comment:
did you run the `go mod tidy` step? i thought this was supposed to mark it with a timestamp after the tag

Reply:
You should only get a timestamp if your upstream isn't released properly or you're targeting a sha. This looks right to me.